jfsd/utils.py

```diff
@@ -717,3 +758,53 @@ def precompute(
     return ((all_indices_x), (all_indices_y), (all_indices_z), gaussian_grid_spacing1, gaussian_grid_spacing2,
             r, indices_i, indices_j, f1, f2, g1, g2, h1, h2, h3,
             r_lub, indices_i_lub, indices_j_lub,ResFunc)
+                                    
+@partial(jit, static_argnums=[3])
+def precomputeBD(
+        positions: float, 
```

Comment:
If it is an array, then it should be an array of floats. If it were a list it would be `list[float]`.

Reply:
Type hints have been updated.